config/initializers/redis.rb

```diff
@@ -1,8 +1,7 @@
 env = AppConfig.env
-ttl = env.service_provider_request_ttl_hours || ServiceProviderRequestProxy::DEFAULT_TTL_HOURS
```

Comment:
looks like this constant isn't used elsewhere, can we remove it?

Reply:
ooh, good catch, removed in 7db3800bf